components/skin/ui/src/main/resources/PhenoTips/Skin.xml

```diff
@@ -794,10 +795,14 @@ h1:target, h2:target, h3:target {
   float: right;
 }
 .language-block .rightmenu a {
-  float: right;
+  color: $theme.menuLinkColor;
   cursor: default !important;
-  text-decoration: blink;
-  padding-top: 0;
+  float: right;
+  font: normal normal 700 0.8em/1.8em sans-serif;
+  margin: 0;
+  padding: 0;
+  text-transform: uppercase;
```

Comment:
We've switched all the other menus from uppercase to normal case, can we drop this line?